### ISSUE TITLE
Add a configuration option to disable SSL for ES connections

### ISSUE THREAD

**Issue by navarone-feekery** [elastic/crawler#149]
### Problem Description

Currently there is no way to run the Crawler over HTTPS without verifying SSL. It's useful to not verify SSL when developing or testing things locally, so there should be an option for this.

### Proposed Solution

Add a configuration option for disabling SSL verification.

### Additional Context

Relevant ES client config:

```
transport_options: {
  ssl: { verify: false }
}
```

Change should be made here: https://github.com/elastic/crawler/blob/45b2999ea440adeb3bd1b3f019a63d74b11f871a/lib/es/client.rb#L149-L162

**Comment by dadoonet** [comment]
Indeed. [The documented](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-compose-file) `docker-compose.yml` generates a self-signed certificate.
On the other hand, may be we should "just" educate the end user to copy the CA fingerprint from the `es01` container (`config/certs/ca/ca.crt`) and copy it to the `crawler` container.
Then use it as the `ca_fingerprint`.

That would transform that "issue" into a documentation issue. 

WDYT?

**Comment by navarone-feekery** [comment]
It is more effort on the user's end to copy the CA fingerprint, but I do like the idea that if the user wants to not verify SSL _without_ the certificate that they will need to manually edit the code.

I'll bring up both ideas internally within the team to see which is preferable.

**Comment by navarone-feekery** [comment]
Reference: we document how to get SSL certs for Connectors, maybe worth adding to Crawler docs: https://github.com/elastic/connectors/blob/main/docs/DOCKER.md#5-run-the-docker-image

**Comment by navarone-feekery** [comment]
We decided that surfacing this as a configuration option is worthwhile so long as we document that it isn't secure and not to use it in production.

**Comment by strawgate** [comment]
I will offer a PR for this by Monday the 7th (or not at all!)
